src/components/shared/GoBackButton.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Button, Col, Container, Row } from "react-bootstrap";
+
+const GoBackButton = (props) => {
+  const onClickHandler = () => {
+    props.onGoBackClick();
+  };
+  return (
+    <Container>
+      <Row>
+        <Col>
+          <Button onClick={onClickHandler}>Go Back</Button>
```

Comment:
Don't pass function as props

Instead of function pass "to" as props

e.g to={"/home"} or to={history.goBack()}